examples/input/mouse_grab.rs

```diff
@@ -19,9 +19,7 @@ fn grab_mouse(
     // There are cases where we can have multiple windows and cursors,
     // but not in this example. Keeping the naming convention since it
     // exists in other examples
-    let Ok((mut window, mut cursor)) = windows_and_cursors.get_single_mut() else {
```

Comment:
it ended up fixing the non-determinism thing here so that we will always have the cursor when windows are created which is perfect